### DISCUSSION TITLE
Minimum required hugo version: community feedback wanted

### DISCUSSION THREAD

**Discussion by deining** [google/docsy#855] [Q&A]
Hi,
I'm currently implementing the transition of docsy theme to a hugo module. In doing so, I have several design choices. However, some of my designs rely on newer hugo versions to make them work. Therefore I'm asking myself: what hugo versions are used out there from the community, especially in production envionments? Are you locked in with this versions or can you easily upgrade? Any input is appreciated, it with be as short as:

````
I'm using hugo 0.73.0 in my production environment, and I'm locked in with that version.
````

Each single feedback answer is appreciated!

**Comment by chalin** [comment]
For reference, #856 is the tracking issue for this topic. I "suggest that we **_officially_** support **only** the release of Hugo specified in `package.json`" -- for details, see in https://github.com/google/docsy/issues/856#issuecomment-1018400541, which is were the statement was quoted from.
